### PR TITLE
Changes the version of postgresql

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Modifying the bundle to use the new [postgresql-k8s](https://github.com/canonical/postgresql-k8s-operator) charm.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: orc8r-bundle/bundle.yaml.j2 [hunk 1]
```diff
@@ -339,7 +339,7 @@ applications:
   postgresql-k8s:
     charm: postgresql-k8s
     series: kubernetes
-    channel: stable
+    channel: edge
```
Comment (gruyaume) [inline orc8r-bundle/bundle.yaml.j2:342]: Do we know if there's a more stable version released somewhere? Example `beta`

Comment (saltiyazan) [reply orc8r-bundle/bundle.yaml.j2:342]: what I see on charmhub is that they only have stable and edge channels :)
https://charmhub.io/postgresql-k8s

Comment (gruyaume) [reply orc8r-bundle/bundle.yaml.j2:342]: @marceloneppel What's the approach to edge? Should we integrate the postgresql-k8s charm in our bundle as is? Do you intend on promoting it to `beta`?

Comment (marceloneppel) [reply orc8r-bundle/bundle.yaml.j2:342]: Hi @gruyaume! Yes, that's the way you should integrate the new charm from edge in the bundle. One thing that I noted is that you have the TLS certificates operator in the bundle. You can also relate the new PostgreSQL from edge to it if you need TLS encrypted communications to the database.

We have plans to promote it to beta, but haven't defined an revision for that yet.

Comment (gruyaume) [reply orc8r-bundle/bundle.yaml.j2:342]: @marceloneppel Yeah I know that's how we integrate the charm from edge but my question is more along the lines of "should we integrate this charm knowing that it's in edge or should we wait for it to be promoted to beta"?

Yes we have the tls-certificates-operator in the bundle but we don't use it for communication with the database. We could use it in the future but not part of this PR.

Comment (marceloneppel) [reply orc8r-bundle/bundle.yaml.j2:342]: Hi @gruyaume! It would be better to wait for it to be promoted to beta, so you don't get any new revision that we're still testing in our PostgreSQL edge bundle. I'm going to discuss with our data team to align the approach/guidelines to promote our charms and promote the PostgreSQL charm to the beta channel soon.

Let me also test the current PostgreSQL charm from edge with this bundle to see how it's working and define the version that can be promoted to beta. Is there a minimal setup that I can use, like a smaller bundle file, just to test the PostgreSQL charm with it?

Comment (gruyaume) [reply orc8r-bundle/bundle.yaml.j2:342]: @marceloneppel ok thank you. We will leave this PR open until postgresql-k8s is published to `beta`. Please let us know the outcome of the team discussion and when we should expect the charm to be promoted.

cc. @saltiyazan

Comment (marceloneppel) [reply orc8r-bundle/bundle.yaml.j2:342]: Hey @gruyaume! We started the discussion yesterday about this topic. We don't have any guidelines yet to promote a charm to a beta or stable channels, but we are starting to define them. It'll probably take some time to align the exact process, but I'll update you here as soon as we have the definition and the charm is promoted to another channel.

Comment (gruyaume) [reply orc8r-bundle/bundle.yaml.j2:342]: Ok thank you @marceloneppel , looking forward to this :)

Comment (gruyaume) [reply orc8r-bundle/bundle.yaml.j2:342]: @marceloneppel is there any update on this? On our side, we'd be perfectly satisfied if the operator was released to `beta` as it currently is.

Comment (marceloneppel) [reply orc8r-bundle/bundle.yaml.j2:342]: Hi @gruyaume! We're almost promoting it to beta. We just upgraded the charm do jammy and are now stabilising some additional things. I'll update here as soon as it's finished.

Comment (gruyaume) [reply orc8r-bundle/bundle.yaml.j2:342]: Thank you @marceloneppel :)

**Comment by saltiyazan** [comment]
The change has been addressed in PR #242 .
Closing this one.
